chaingpt-research.md

```diff
@@ -41,17 +39,95 @@ Consider using Ubiquity OS as a launchpad of plugins and other products for deve
 - Alex reviews user feedback on ChainGPT.
 This may provide new ideas for leveraging ChainGPT's experience.
 
-**Key actions that can be relevant to us:**
+### Key actions that can be relevant to us
+
+**1. Establish cooperation with an effective crypto marketing agency**
+
+It is known from open sources that ChainGPT used several marketing agencies.
+In reality, there were probably more agencies than are mentioned in open sources.
+
+- Lead marketing agency - Coinband
+
+https://coinband.io/case/chaingpt  
+
+- Website design - Sigma Software 
+
+https://sigma.software/about/media/a-winning-streak-for-our-design-teams-new-project-chaingpt
+
+https://dribbble.com/shots/23245957-ChainGPT-Website-Behance-Case-Study  
+
+https://www.behance.net/gallery/186506369/ChainGPT-Blockchain-AI-Website 
+
+- One of the additional marketing agencies - EmoneyMax
+
+https://emoneymax.net/  
+
+- ChainGPT itself provides marketing services in the scope of the ChainGPT Pad 
+
+https://medium.com/@chaingpt/chaingpt-pad-2023-a-year-of-unprecedented-success-in-web3-11e7db93615c 
+
+Finding other effective agencies that are on the market but have not been used by ChainGPT can be done in separate research.
+
+
+**2. Clearly define the main narrative that we bring to the market**
+
+Alex: will do when producing Series A fundraising materials.
```

Comment:
I need to spend more time copying (for a lack of better term) what other crypto AI startups say. You put together some interesting narratives of the other projects in another thread somewhere. When I catch more time to work on the pitchdeck, I'll spend a lot of time studying the other narratives. 

Please feel free to follow along on the pitchdeck progress at https://github.com/ubiquity/series-a/issues/13

Please feel free to leave comments, or fork a copy (don't edit in place!) if you're inspired to edit the deck.